README.md

```diff
@@ -9,12 +9,14 @@ same Lead.
 All configuration can be done in the form generator from Contao. Additionally you can set a label 
 for the backend module of your Lead and define the listing of the form data using simple tags.
 
-The leads extension offers additionally an export function (CSV and  - you need to install the
-"excel" extension) for each Lead in the backend. You can configure it as you wish!
+The leads extension additionally offers an export function for each Lead in the backend. You can 
```

Comment:
The docs are not wrong. In the ER it's "excel" :-) Can you mention both, maybe? :-)